Процедура НегативныйТест1()
    ПродолжитьВЫЗОВ(); // <- Ошибка, вызов в методе без аннотации &Вместо
КонецПроцедуры

Процедура НегативныйТест2(Парам1)
    ProceedWithCall(Парам1); // <- Ошибка, вызов в методе без аннотации &Вместо
КонецПроцедуры

&Перед(НегативныйТест3)
Функция Расш_НегативныйТест3(Парам1, Парам2)
    Перем1 = ПродолжитьВызов(Парам1, Парам2); // <- Ошибка, вызов в методе без аннотации &Вместо
    Возврат Перем1;
КонецФункции

&После(НегативныйТест4)
Функция Расш_НегативныйТест4(Парам1, Парам2)
    Перем1 = ProceedWithCall(Парам1, Парам2); // <- Ошибка, вызов в методе без аннотации &Вместо
    Возврат Перем1;
КонецФункции

Процедура А()
    ПродолжитьВызовОбработчика();
    _ПродолжитьВызов();
    Модуль.ПродолжитьВызовОбработчика();
    Модуль._ПродолжитьВызов();
КонецПроцедуры

&Вместо(ПозитивныйТест1)
Процедура Расш_ПозитивныйТест1()
    ПродолжитьВЫЗОВ();
КонецПроцедуры

&Вместо(ПозитивныйТест2)
Процедура Расш_ПозитивныйТест2(Парам1)
    ProceedWithCall(Парам1);
КонецПроцедуры

&Вместо(ПозитивныйТест3)
Функция Расш_ПозитивныйТест3(Парам1, Парам2)
    Перем1 = ПродолжитьВызов(Парам1, Парам2);
    Возврат Перем1;
КонецФункции

&Вместо(ПозитивныйТест4)
Функция Расш_ПозитивныйТест4(Парам1, Парам2)
    Перем1 = ProceedWithCall(Парам1, Парам2);
    Возврат Перем1;
КонецФункции
